### PR TITLE
doc: add meeting notes for 07-Apr-2016 meeting

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

File: meetings/2016-04-07.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Use The Foundation’s YouTube Channel 
+
+- https://github.com/nodejs/inclusivity/issues/107 
+- Bryan: Resolved! This meeting is on the Foundation’s YouTube Channel.
+
+## Create a history document
+
+- https://github.com/nodejs/inclusivity/issues/85
+- Bryan: Mostly done, but there are a few nits/issues. One is that there is a flat list of members in the README, but in the history page, we list when people are added. Right now, those lists are in sync, but they may not be. In one they are called, “Members”, but in the other it says “Initial Membership”. Those sorts of things will hopefully get ironed out. William K. is active in that issue.
```
Comment (scottgonzalez) [inline meetings/2016-04-07.md]: Drop the comma before "Members"

File: meetings/2016-04-07.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- Scott: “Current Members” should be title in README. For history doc, list all current and previous members in one list, or have two lists (one for current and one for previous). It might make sense to just do the one list because we already have the list of active users in README.
+- Julie: I concur. We can split it up later if we want to.
+- Bryan: I’m wary of singling out people who are no longer members.
+- Julie: That’s a really good point. Let’s just have one list in the history doc. No need to split them up.
+- Bryan: It is still possible to figure it out. But it takes some works.
```
Comment (scottgonzalez) [inline meetings/2016-04-07.md]: s/works/work/

**Comment by 59** [comment]
LGTM 👍

File: meetings/2016-04-07.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- Scott: If there’s one list, we don’t take an action when someone becomes a past member. But if we have two lists, there will be notifications and stuff from PRs, etc.
+- Bryan: Good point. There will still be notification from the PR for the README, though.
+- Scott: Settled then? We’ll just change the title on the README to say “Current Members” and the history do for “Current and Past Members”.
```
Comment (scottgonzalez) [inline meetings/2016-04-07.md]: s/do/doc/ in "the history do for"

**Comment by nebrius** [comment]
Once @scottgonzalez nits are fixed, I think this'll be ready to land

File: meetings/2016-04-07.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+## Create page for community members to join our Slack team
+
+- https://github.com/nodejs/inclusivity/issues/111
+- Scott: The goal is to enable public access to the Slack channel. We are using an undocumented Slack API to invite members. Slack acknowledges that people use this, and they make no promises about it, although they won’t break it on purpose. For now it works, and others are using it. The PR has the bare minimum implementation. It gives you access to our public channels. GitHub account needs to exist for 30 days to make it harder to create throwaway accounts to join and cause trouble.
+- Bryan: We are intentionally linking this to GitHub authentication so we can take moderation activity in the GitHub repos too if we need to.
+- Julie: Sounds good to me. As long as we have language that says “If you have a problem, here’s how you get in touch with us.”
+- Bryan: I was going to include the inclusivity email.
+- Jona: They can open an issue on the GitHub repo.
+- Bryan: Good point. I hadn’t considered that. Although then it is a public request rather than a private request.
+- Julie: Yeah, I don’t want to make people file a GitHub issue if they dont’ want to.
```
Comment (scottgonzalez) [inline meetings/2016-04-07.md]: s/dont'/don't/

Comment (59) [inline meetings/2016-04-07.md]: DAMMIT

**Pull request merged**